### ISSUE TITLE
Brushing up the default look of theme

### ISSUE THREAD

**Issue by paaljoachim** [wplemon/gridd#91]
I made a new dev site and added Jarvis and Gridd. After having played a bit with Jarvis I then went to test Gridd. Here is the default look after having turned on Gridd:

![Screen Shot 2019-08-24 at 12 47 07](https://user-images.githubusercontent.com/5323259/63636253-53224300-c66d-11e9-8d37-1a59b96a1642.png)

What I experience is:
A very tall header taking up a lot of space. With a menu top right and the title Multisite lower down and up against the left edge. 
Some text on the front page.
Footer border not connected with copyright info.

I would like to see a default that pushes the site title further up and a little more inward on the page. Probably on the same line as the menu. Here is a suggestion. I also added a Top link bottom right (could also be the top link icon).

![new-Gridd-theme-default-look](https://user-images.githubusercontent.com/5323259/63636546-f58ff580-c670-11e9-89d2-dff08e52584e.jpg)

The image brings symmetry between various elements on the page. Site title and menu are on a line. Site title and copyright info are also on a line. I brought the blue underline closer to the menu item and made it smaller to fit with the Front menu link. Bringing the blue line closer adds a relation between the link and the blue line.

Copyright info and top are also aligned as so is the menu and top link. I would also suggest that you just add the top link text to the free version and have some more options in regards to top link in the premium version. As Back to top link is a fairly common feature.

Bottom line is that it would be great to brush up the initial look, feel and symmetry of the initial default Gridd theme experience.

**Comment by aristath** [comment]
I agree, when you first activate the gridd theme it's ugly. The problem here is finding some sane defaults that would work for most cases without making a lot of assumptions about the way a user would want their site. Right now when you activate it it's almost a blank canvas - which has its pros and cons.
If you have suggestions on some default values that would work and would make things prettier I'm open to suggestions 😄

**Comment by paaljoachim** [comment]
New default CSS.

Header Grid.
Branding - Padding: 0 0 0 1.7em
Navigation - 0.3em 0 0 1em

Global Site Grid.
Header - changed from auto to 90 px. To make the header smaller. Auto should also automatically make it a lot smaller then the default of what it is today.

Footer - changed from auto to 90px. To make the footer smaller. Auto should also automatically make it a lot smaller then the default of what it is today. When widgets are added then the footer becomes wider to reflect that widgets are present.

Footer Grid - Not able to adjust the padding as I could in the header. Move copyright line a little up. I am making a new issue comparing the Header and Footer grid as these really should be almost identical in the options they offer.


--

Move the underline closer up to the menu link. (What would the CSS be for this?) So both are more connected.

In general just take a look at the image I made in Photoshop by cutting and realigning various areas of the layout.

**Comment by carolinan** [comment]
I think the menu and submenus could use a tiny bit of freshening up as well.

- When you hover over, or focus on a menu item that has submenus, or on the arrow itself, 
the position of the arrow changes, because the default state for the arrow does not have a bottom border. When the bottom border is applied, the arrow jumps upwards. This is an easy fix.


![arrow](https://user-images.githubusercontent.com/7422055/63645079-20249180-c6f7-11e9-951c-102ba41cafbe.gif)



- When you have a long menu label and more than one submenu level, the toggle/arrow is no longer aligned to the right. When _toggled_, the arrow moves and is aligned to the right as expected.

![depth](https://user-images.githubusercontent.com/7422055/63645061-c15f1800-c6f6-11e9-8cb0-4592b2774cf6.gif)

**Comment by aristath** [comment]
> When you hover over, or focus on a menu item that has submenus, or on the arrow itself, the position of the arrow changes, because the default state for the arrow does not have a bottom border. When the bottom border is applied, the arrow jumps upwards. This is an easy fix.

Fixed

**Comment by aristath** [comment]
> When you have a long menu label and more than one submenu level, the toggle/arrow is no longer aligned to the right. When toggled, the arrow moves and is aligned to the right as expected.

Good find! Fixed.

**Comment by aristath** [comment]
Closing this one. Some default styles were tweaked in the develop branch and more will follow.
